x-pack/plugins/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -16,25 +17,39 @@ import { useLocalStorage } from '../../../hooks/useLocalStorage';
 import { FETCH_STATUS, useFetcher } from '../../../hooks/use_fetcher';
 import { useUpgradeAssistantHref } from '../../shared/Links/kibana';
 import { SearchBar } from '../../shared/search_bar';
+import { getTimeRangeComparison } from '../../shared/time_comparison/get_time_range_comparison';
 import { NoServicesMessage } from './no_services_message';
 import { ServiceList } from './service_list';
 import { MLCallout } from './service_list/MLCallout';
 
 const initialData = {
-  items: [],
-  hasHistoricalData: true,
-  hasLegacyData: false,
+  requestId: '',
+  mainStatistics: { items: [], hasHistoricalData: true, hasLegacyData: false },
 };
 
 let hasDisplayedToast = false;
 
 function useServicesFetcher() {
   const {
-    urlParams: { environment, kuery, start, end },
+    urlParams: {
+      environment,
+      kuery,
+      start,
+      end,
+      comparisonEnabled,
+      comparisonType,
+    },
   } = useUrlParams();
   const { core } = useApmPluginContext();
   const upgradeAssistantHref = useUpgradeAssistantHref();
 
+  const { offset } = getTimeRangeComparison({
+    start,
+    end,
+    comparisonEnabled,
+    comparisonType,
+  });
+
   const { data = initialData, status } = useFetcher(
```

Comment:
Perhaps rename these to align with the comparison `data` and `status`

```suggestion
  const { data: mainStatisticsData = initialData, status: mainStatisticsStatus } = useFetcher(
```